mill-scalafix/src/com/goyeau/mill/scalafix/ScalafixModule.scala

```diff
@@ -15,6 +15,23 @@ import scala.jdk.CollectionConverters.*
 trait ScalafixModule extends ScalaModule {
   def scalafixConfig: T[Option[os.Path]] = T(None)
   def scalafixIvyDeps: T[Agg[Dep]]       = Agg.empty[Dep]
+
+  /** Override this to filter out repositories that don't need to be passed to scalafix
+    *
+    * Repositories passed to scalafix need to be converted to the coursier-interface API (coursierapi.*). This can be an
+    * issue for non-Maven or Ivy repositories. Overriding this task and filtering those repositories out allows to work
+    * around that.
+    */
+  def scalafixRepositories: Task[Seq[Repository]] = Task.Anon {
```

Comment:
What's the difference between `Task.Anon` and `Task`?

Reply:
The former doesn't serialize and cache its result, while the latter does. `CoursierModule#repositoriesTask` relies on that for example, as all `coursier.Repository` instances cannot all be serialized to JSON (`coursier.Repository` is an "open" abstract class, we don't know all its implementations beforehand). We need to do the same here, as `coursier.Repository` is involved.